### PR TITLE
Deprecation of org scoped promoted results in modern

### PR DESCRIPTION
**Pull request by wobba**
Added an important note to highlight that promoted results by query rules in modern for org scoped queries are being deprecated in December. Message center post MC293777 https://admin.microsoft.com/?&source=applauncher&ref=MessageCenter/:/messages/MC293777

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/manage-query-rules.md [hunk 1]
```diff
@@ -36,7 +36,8 @@ You can configure query rules for one or more result sources, and you can specif
 
 SharePoint has both a classic and a modern search experience. Only query rules that return promoted results and that are defined for the default result source can affect the modern search experience. Query rules which do query re-writes are not supported for modern experiences. Users might see such promoted results on the **All** tab on the search results page when they search across all of SharePoint. [Learn more about the differences between the classic and modern search experiences in SharePoint](differences-classic-modern-search.md).
   
- 
+> [!IMPORTANT]
+> In December 2021 query rules with promoted results are deprecated for organization scoped queries in favor of [Bookmark and Q&A answers](https://docs.microsoft.com/en-us/microsoftsearch/manage-bookmarks) in modern search experiences. Examples of organization scoped queries are SharePoint Home and office.com. <p>Promoted results will continue to work for all classic search experineces and for sites and hub sites with modern search experiences until Bookmark answers are available for these scenarios. Promoted results will also continue to work for environments where Bookmark and Q&A answers are not available.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-query-rules.md:40]:
```suggestion
> In December 2021, query rules with promoted results are deprecated for organization scoped queries in favor of [Bookmark and Q&A answers](/microsoftsearch/manage-bookmarks) in modern search experiences. Examples of organization scoped queries are SharePoint Home and office.com. <p>Promoted results will continue to work for all classic search experiences and for sites and hub sites with modern search experiences until Bookmark answers are available for these scenarios. Promoted results will also continue to work for environments where Bookmark and Q&A answers are not available.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @wobba, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Pull request merged**